client/components/select-dropdown/style.scss

```diff
@@ -216,6 +225,11 @@ $compact-header-height: 35;
 			color: $white;
 		}
 	}
+
+	.gridicon {
+		margin-right: #{ ( $side-margin / 4 ) }px;
```

Comment:
Sounds good. I had gone with 4px to move it closer in examples where `count` is defined too, but in the initial integration had used 8px. Should've gone with my gut :)